crates/sui-indexer-alt-graphql/src/api/query.rs

```diff
@@ -432,6 +433,26 @@ impl Query {
         }
     }
 
+    /// Range of checkpoints for which data is available for a query type, field and optional filter.
```

Comment:
We should also document what happens if the `filter` is omitted but the query supports filters.

Reply:
Oh, also, this should be under `ServiceConfig`, rather than on the top-level `Query`.